queryparams/params.go

```diff
@@ -367,7 +371,10 @@ const (
 	Postponed
 )
 
-var relTypeNames = map[ReleaseType]string{Upcoming: "type-upcoming", Published: "type-published", Cancelled: "type-cancelled", Provisional: "subtype-provisional", Confirmed: "subtype-confirmed", Postponed: "subtype-postponed", InvalidReleaseType: "Invalid"}
+var (
+	relTypeNames     = map[ReleaseType]string{Upcoming: "type-upcoming", Published: "type-published", Cancelled: "type-cancelled", Provisional: "subtype-provisional", Confirmed: "subtype-confirmed", Postponed: "subtype-postponed", InvalidReleaseType: "Invalid"}
+	relTypeIcalNames = map[ReleaseType]string{Upcoming: "Upcoming", Published: "Published", Cancelled: "Cancelled", Provisional: "Provisional", Confirmed: "Confirmed", Postponed: "Postponed", InvalidReleaseType: "Invalid"}
```

Comment:
Could we use a single map with a struct type as values instead? I'd say it would be easier to maintain and read?

Reply:
will do